docs/getting-started-guides/kubeadm.md

```diff
@@ -258,127 +301,77 @@ should only install one pod network per cluster.
-[tokens] Validating provided token
-[discovery] Created cluster info discovery client, requesting info from "http://192.168.x.y:9898/cluster-info/v1/?token-id=f11877"
-[discovery] Cluster info object received, verifying signature using given token
-[discovery] Cluster info signature and contents are valid, will use API endpoints [https://192.168.x.y:6443]
-[bootstrap] Trying to connect to endpoint https://192.168.x.y:6443
-[bootstrap] Detected server version: v1.5.1
-[bootstrap] Successfully established connection with endpoint "https://192.168.x.y:6443"
+[discovery] Trying to connect to API Server "10.138.0.4:6443"
+[discovery] Created cluster-info discovery client, requesting info from "https://10.138.0.4:6443"
+[discovery] Cluster info signature and contents are valid, will use API Server "https://10.138.0.4:6443"
+[discovery] Successfully established connection with API Server "10.138.0.4:6443"
+[bootstrap] Detected server version: v1.6.0-beta.3
+[bootstrap] The server supports the Certificates API (certificates.k8s.io/v1beta1)
 [csr] Created API client to obtain unique certificate for this node, generating keys and certificate signing request
-[csr] Received signed certificate from the API server:
-Issuer: CN=kubernetes | Subject: CN=system:node:yournode | CA: false
-Not before: 2016-12-15 19:44:00 +0000 UTC Not After: 2017-12-15 19:44:00 +0000 UTC
-[csr] Generating kubelet configuration
+[csr] Received signed certificate from the API server, generating KubeConfig...
 [kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
 
 Node join complete:
 * Certificate signing request sent to master and response
-    received.
+  received.
 * Kubelet informed of new secure connection details.
 
 Run 'kubectl get nodes' on the master to see this machine join.
 ```
 
-A few seconds later, you should notice that running `kubectl get nodes` on the
-master shows a cluster with as many machines as you created.
+A few seconds later, you should notice this node in the output from `kubectl get
+nodes` when run on the master.
 
 ### (Optional) Controlling your cluster from machines other than the master
 
-In order to get a kubectl on your laptop for example to talk to your cluster,
-you need to copy the `KubeConfig` file from your master to your laptop like
-this:
+In order to get a kubectl on some other computer (e.g. laptop) to talk to your
+cluster, you need to copy the kubeconfig file from your master to your
+workstation like this:
 
 ``` bash
 scp root@<master ip>:/etc/kubernetes/admin.conf .
 kubectl --kubeconfig ./admin.conf get nodes
 ```
 
+**Note:** By default GCE instances disable ssh access for root.  First log in to
+the machine, copy the file someplace that can be accessed and then use [`gcloud
+compute
+copy-files`](https://cloud.google.com/sdk/gcloud/reference/compute/copy-files)
+
 ### (Optional) Connecting to the API Server
 
 If you want to connect to the API Server for viewing the dashboard (note: the
```

Comment:
let's just get rid of the reference to dashboard here, if we're not going to describe how to install it

"If you want to connect to the API Server from outside the cluster you can use"